setup.py

```diff
@@ -23,7 +23,7 @@
         "web3",
         "gql[requests]",
         "requests",
-        "bal_tools @ git+https://github.com/BalancerMaxis/bal_tools@v0.0.1",
+        "bal_tools @ git+https://github.com/BalancerMaxis/bal_tools@v0.0.2",
```

Comment:
This seems to be working, but never seen a space there before and the "s.  Some day tell me what that is about.

Reply:
as far as i understand this is just a different notation from `requirements.txt`.

tbh, using `setup.py` is a bit outdated and should be replaced with poetry's `pyproject.toml` imo